src/EnergyPlus/HVACMultiSpeedHeatPump.hh

```diff
@@ -64,19 +64,14 @@ struct EnergyPlusData;
 
 namespace HVACMultiSpeedHeatPump {
 
-    // Using/Aliasing
-
-    // Data
-    // MODULE PARAMETER DEFINITIONS
-
     // Heating coil types
-    extern int const MultiSpeedHeatingCoil; // COIL:DX:MultiSpeed:Heating
+    int constexpr MultiSpeedHeatingCoil(1); // COIL:DX:MultiSpeed:Heating
     // Cooling coil types
-    extern int const MultiSpeedCoolingCoil; // COIL:DX:MultiSpeed:Cooling
+    int constexpr MultiSpeedCoolingCoil(2); // COIL:DX:MultiSpeed:Cooling
     // Supplymental heating coil types
-    extern int const SuppHeatingCoilGas;  // Supplymental heating coil type: COIL:GAS:HEATING
-    extern int const SuppHeatingCoilElec; // Supplymental heating coil type: COIL:ELECTRIC:HEATING
-    extern int const SuppHeatingCoilRec;  // Supplymental heating coil type: COIL:ENGINEHEATRECOVERY:HEATING
+    int constexpr SuppHeatingCoilGas(1);  // Supplymental heating coil type: COIL:GAS:HEATING
+    int constexpr SuppHeatingCoilElec(2); // Supplymental heating coil type: COIL:ELECTRIC:HEATING
+    int constexpr SuppHeatingCoilRec(3);  // Supplymental heating coil type: COIL:ENGINEHEATRECOVERY:HEATING
```

Comment:
`constexpr` is used instead of `enum class` since these values are compared with integer values of parameters that are describing coil types in DataHVACGlobals for now.